### DISCUSSION TITLE
挺不错的组件库

### DISCUSSION THREAD

**Discussion by elarbor** [FriedRiceNoodles/banana-ui#174] [General]
终于找到了 `  Web Components ` 的组件库。看了看，组件稍微有点少，希望未来能完善。

**Comment by FriedRiceNoodles** [comment]
感谢认可，目前组件库的内容大多是基于目前我所供职的公司的业务需求开发出来并完善的，虽然组件数量较少但是都有经受过业务的考验；当然也十分欢迎你作出贡献，开发新的组件或是其他任何形式的贡献都是极好的。
如果你对为Banana开发新的组件有想法，仓库内实际上有一个用于开发新组件的脚手架，可以帮助你快速启动。你可以试着在项目的根目录执行`pnpm banana new`(前置条件是在项目根目录执行过`pnpm install`)，来为Banana新建一个组件。

**Reply by heiyanquan** [reply]
问一下，你们的css都是怎么写出来的，我看了一下vscode倒是有个lit-plugin插件可以高亮ts文件里的css，并且书写过程中有提示，还有其他更好的办法吗？非常好奇你们的css都是怎么写出来的，一般不可能纯手撸不用提示吧

**Reply by elarbor** [reply]
用unocss

**Reply by FriedRiceNoodles** [reply]
> 问一下，你们的css都是怎么写出来的，我看了一下vscode倒是有个lit-plugin插件可以高亮ts文件里的css，并且书写过程中有提示，还有其他更好的办法吗？非常好奇你们的css都是怎么写出来的，一般不可能纯手撸不用提示吧

涉及到开发环境层面的事情，只能在开发环境解决。暂时没找到比用lit提供的vscode拓展更好的办法，不过至少在我的开发过程来说它还算是够用的。

**Reply by FriedRiceNoodles** [reply]
> 问一下，你们的css都是怎么写出来的，我看了一下vscode倒是有个lit-plugin插件可以高亮ts文件里的css，并且书写过程中有提示，还有其他更好的办法吗？非常好奇你们的css都是怎么写出来的，一般不可能纯手撸不用提示吧

另外，由于lit给出的样式开发方式是css\`.......\`这样的形式，也就是'css' + 模板字符串，你可以找到不少跟lit不相关，但是可用的模板字符串增强拓展。

**Reply by heiyanquan** [reply]
好的，谢谢！
